src/linagora.esn.controlcenter/app/app.module.js

```diff
@@ -1,3 +1,16 @@
+require('esn-frontend-common-libs/src/frontend/js/modules/esn.router.js');
+require('esn-frontend-common-libs/src/frontend/js/constants.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/header/header.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/user/user.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/sidebar.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/module-registry/module-registry.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/user-configuration/user-configuration.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/business-hours/business-hours.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/config/config.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/i18n/i18n.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/home-page/home-page.module.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/application-menu.js');
+
```

Comment:
So, you must `require` all the local files from this `linagora.esn.account` module